### PR TITLE
Add mobi.snowball.Snowball

### PR DESCRIPTION
**Pull request by BeverlyCode**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

File: mobi.snowball.Snowball.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+app-id: com.voxdsp.Snowball
```
Comment (hfiguiere) [inline mobi.snowball.Snowball.yaml:1]: manifest and this don't match

File: mobi.snowball.Snowball.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    build-commands:
+      - cc main.c glad_gl.c -Ofast -L/app/lib -lglfw -lm -o snowball
+      - install -Dm 0755 snowball -t ${FLATPAK_DEST}/bin
+      - install -Dm 0644 flatpak/snowball.desktop ${FLATPAK_DEST}/share/applications/snowball.desktop
+      - install -Dm 0644 flatpak/snowball.appdata.xml ${FLATPAK_DEST}/share/metainfo/snowball.appdata.xml
+      - install -Dm 0644 flatpak/snowball.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/snowball.png
+    sources:
+      - type: git
+        url: https://notabug.org/Vandarin/Snowball.git
+        commit: 000
```
[the remark this answers is not in the document]
Comment (hfiguiere) [reply mobi.snowball.Snowball.yaml:39]: this is an invalid commit sha

**Comment by hfiguiere** [comment]
bot, build mobi.snowball.Snowball

**Comment by hfiguiere** [comment]
```
FB: Mounting read-only fs: rofiles-fuse /srv/buildbot/worker/build-x86_64-5/build/builddir /srv/buildbot/worker/build-x86_64-5/build/.flatpak-builder/rofiles/rofiles-9yAUvw
Renaming snowball.appdata.xml to share/appdata/mobi.snowball.Snowball.appdata.xml
Renaming snowball.desktop to mobi.snowball.Snowball.desktop
Error: Error on line 18 char 16: Element “description” was closed, but the currently open element is “p”
```

**Comment by bbhtt** [comment]
bot, build mobi.snowball.Snowball

**Comment by bbhtt** [comment]
`fatal: unable to access 'https://notabug.org/Vandarin/Snowball.git/': The requested URL returned error: 504`

**Comment by bbhtt** [comment]
bot, build mobi.snowball.Snowball

**Pull request closed without merging**